src/pages/TestPage/TestPage.jsx

```diff
@@ -1,20 +1,34 @@
 import "./TestPage.scss";
+<<<<<<< HEAD
+import { useEffect, useState } from "react";
+import Modal from "react-modal";
+import { Button } from "@mui/material";
+import { PopUpModal } from "../../components/PopUpModal/PopUpModal";
+import { getAllJobsData } from "../../utils/Functions/functions";
+=======
 import Button from "@mui/material/Button";
 import { useState } from "react";
 import Modal from "react-modal";
 import { PopUpModal } from "../../components/PopUpModal/PopUpModal";
+>>>>>>> 3c2c5829efbb1817eefd755f8a15c335bce18236
 import LoadingPage from "../LoadingPage/LoadingPage";
 
 const TestPage = () => {
   const [isModalOpen, setModalOpen] = useState(false);
+  const [jobs, setJobs] = useState("");
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const result = await getAllJobsData("Jobs");
+      setJobs(result[0]);
+      console.log(result[0]);
+    };
+    fetchData();
```

Comment:
Case in point you define it just to call it here. Why don't you just put the logic into the use effect without defining it as a function